threeML/plugins/FermipyLike.py

```diff
@@ -26,23 +26,22 @@
 
 
 # A lookup map for the correspondence between IRFS and evclass
-evclass_irf = {2: 'P8R2_TRANSIENT100E_V6',
-               4: 'P8R2_TRANSIENT100_V6',
-               8: 'P8R2_TRANSIENT020E_V6',
-               16: 'P8R2_TRANSIENT020_V6',
-               32: 'P8R2_TRANSIENT010E_V6',
-               64: 'P8R2_TRANSIENT010_V6',
-               128: 'P8R2_SOURCE_V6',
-               256: 'P8R2_CLEAN_V6',
-               512: 'P8R2_ULTRACLEAN_V6',
-               1024: 'P8R2_ULTRACLEANVETO_V6',
+evclass_irf = {2    : 'P8R2_TRANSIENT100E_V6',
```

Comment:
ah ah! you have a strange taste... ;-)
Anyway, please refrain from doing all these small changes in pull requests because it makes the number of files I need to check very large... I authorize you to make aesthetic changes like these directly in the trunk :-)